### ISSUE TITLE
Cannot restore, refresh keys or send funds

### ISSUE THREAD

**Issue by joel-smalley** [kenCode-de/smartcoins-wallet#61]
Hi,

Since registering my Bitshares account on the Smartcoin app, I have lost complete control. The backup.bin does not import into the Bitshares app and the brainkey backup does not import the private key so I can no longer operate my Bitshares account in the Bitshares app (web or desktop). 

I cannot refresh the key in the Smartcoin app (error: "It was not possible to update your account key this time. Please check your balance.") and I cannot send any of my funds to another account ("An error occurred while broadcasting your transaction.")

I am running Android 6 on a Sony Xperia Ultra UA F3216.

**Comment by bilthon** [comment]
Please be careful with the "key refreshing" functionality, you should write down the newly provided brainkey once the procedure is done.

The bin backup is not compatible with either the web wallet or the desktop app, and that is a known issue that we're working on. But having your correct brain key will always give you access to your account.

If you provide us with your account name information, we can try to better diagnose the problem. If you don't wish to provide it here at a public github issue you can send me an email to bilthon@gmail.com.

**Comment by kenCode-de** [comment]
Hi @joel-smalley I just wanted you to know that your Issue will be fixed in roughly one week from now and we will release an app update with the fix, and then close this Issue here which should notify you automatically so that you can grab the download. Sorry for the trouble, and rest assured this issue is a top priority.

**Comment by joel-smalley** [comment]
Thank you! Joel SmalleyManaging DirectorWeb: http://supercapital.ukSupercapital Ltd, 3 Beeston Place, London, SW1W 0JJ. Incorporated in England & Wales no. 04306276 Authorised & Regulated by the FCA for PSR 2009 no. 662541 and HMRC for MLR 2007 no. 12752676On 10 February 2017 at 08:33 ken Code <notifications@github.com> wrote:Hi @joel-smalley I just wanted you to know that your Issue will be fixed in roughly one week from now and we will release an app update with the fix, and then close this Issue here which should notify you automatically so that you can grab the download. Sorry for the trouble, and rest assured this issue is a top priority.—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by joel-smalley** [comment]
Hi,

Is this issue fixed now? Please tell me what I need to do!

Thanks, Joel

**Comment by kenCode-de** [comment]
Hi Joel, if you have a good backup of your .bin file, Brainkey and PIN number, then it might be best if you totally uninstall the app, then reinstall the latest version of Smartcoins Wallet from google play. As always, never put more into your wallet than you can afford to lose. I think Nelson was helping you with this, right? Since you posted the issue on github, he has added support for web wallet and light client wallet bin file importing and exporting too.

**Comment by kenCode-de** [comment]
By the way, which bitshares account name was it?

**Comment by joel-smalley** [comment]
Yes, that's right. I'm not sure I do have good backups of all those files (I don't really know how to do so and certainly don't have all the Brainkeys which Nelson said I might need to find out exactly what's what?). He advised not to uninstall anything. Shall I just wait for Nelson to revert on Telegram?Regards, Joel Joel SmalleyManaging DirectorWeb: http://supercapital.ukSupercapital Ltd, 3 Beeston Place, London, SW1W 0JJ. Incorporated in England & Wales no. 04306276 Authorised & Regulated by the FCA for PSR 2009 no. 662541 and HMRC for MLR 2007 no. 12752676On 15 March 2017 at 11:23 ken Code <notifications@github.com> wrote:Hi Joel, if you have a good backup of your .bin file, Brainkey and PIN number, then it might be best if you totally uninstall the app, then reinstall the latest version of Smartcoins Wallet from google play. As always, never put more into your wallet than you can afford to lose. I think Nelson was helping you with this, right? Since you posted the issue on github, he has added support for web wallet and light client wallet bin file importing and exporting too.—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by joel-smalley** [comment]
super-joel Joel SmalleyManaging DirectorWeb: http://supercapital.ukSupercapital Ltd, 3 Beeston Place, London, SW1W 0JJ. Incorporated in England & Wales no. 04306276 Authorised & Regulated by the FCA for PSR 2009 no. 662541 and HMRC for MLR 2007 no. 12752676On 15 March 2017 at 11:24 ken Code <notifications@github.com> wrote:By the way, which bitshares account name was it?—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.

**Comment by kenCode-de** [comment]
oh, yeah I see you created that account with their OpenLedger wallet: 
http://cryptofresh.com/u/super-joel 
 
yeah ok then you should wait until Nelson @bilthon returns from vacation on monday and you guys work out how you want to proceed.
